src/bmson.rs

```diff
@@ -672,3 +675,245 @@ impl TryFrom<Bms> for Bmson {
         })
     }
 }
+
+impl From<Bmson> for Bms {
+    fn from(value: Bmson) -> Self {
```

Comment:
This process isn't considered about it is lossy (loses precise data) conversion for Bmson. Even thought parsing BMS from the source text produces warning, but this function ignores all conflicts of insertions.
I think it should be changed into a method like `from_bmson_lossy` that returns warnings too.